Enhancement: Run "pa app get" as admin. Closes #5518 (#5609)
pnp/cli-microsoft365: docs/docs/cmd/pa/app/app-get.mdx

`-d, --displayName [displayName]`
: The display name of the Microsoft Power App to get information about.

`-e, --environmentName [environmentName]`
: The name of the environment for which to retrieve the specified app.

> `--asAdmin`

Reviewer: Let's put this before `environmentName`.

:::

> As maker, you are able to retrieve the Power Apps you own. As administrator, you are also able to retrieve Power Apps from other users. To get the app from other user, use the `asAdmin` option and make sure to specify the `environment` option. You cannot specify only one of the options, when specifying the `environment` option the `asAdmin` option has to be present as well.

Reviewer: ```suggestion
As a maker, you are able to retrieve the Power Apps you own. As administrator, you are also able to retrieve Power Apps from other users. To get the app from another user, use the `asAdmin` option and make sure to specify the `environment` option as well.
```

m365 pa app get --displayName App
```

> Get information about the specified Microsoft Power App from a given environment by the app's name.

Reviewer: ```suggestion
Get information about the specific app you do not have access to.
```
